Mono.Cecil/GenericInstanceType.cs

```diff
@@ -26,7 +26,7 @@ public sealed class GenericInstanceType : TypeSpecification, IGenericInstance, I
 		}
 
 		public Collection<TypeReference> GenericArguments {
-			get { return arguments ?? (arguments = new Collection<TypeReference> ()); }
+			get { return arguments ?? (arguments = new Collection<TypeReference> (4)); }
```

Comment:
Is this necessary? The new Collection will be using `Empty<TypeReference>.Array` as it starts empty which is already allocated, and it will grow to `4`, the default capacity, when needed.

Reply:
you're right, reverted.